src/main/java/de/hpi/bpt/argos/persistence/database/DatabaseConnectionImpl.java

```diff
@@ -108,28 +111,13 @@ public ProductFamily getProductFamily(long productFamilyId) {
 	@Override
 	public Product getProduct(long productId) {
 		Session session = databaseSessionFactory.openSession();
-		Transaction tx = null;
-		Query<Product> query = null;
-		try {
-			tx = session.beginTransaction();
-			query = session.createQuery("FROM ProductImpl p "
-							+ "WHERE p.id = :productId",
-					Product.class)
-					.setParameter("productId", productId);
+		Transaction transaction = session.beginTransaction();
+		Query<Product> query = session.createQuery("FROM ProductImpl p "
+						+ "WHERE p.id = :productId",
+				Product.class)
+				.setParameter("productId", productId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "productId" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)